src/components/uiOptions/js/UIOptions.js

```diff
@@ -19,7 +19,8 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     });
 
     fluid.defaults("fluid.uiOptions.prefsEditor", {
-        gradeNames: ["fluid.prefs.constructed.prefsEditor"],
+        gradeNames: ["fluid.prefs.constructed.prefsEditor", "{that}.getLazyLoadGrade"],
+        lazyLoad: false,
```

Comment:
This can/should be done via the contextAwareness facility. It's much more idiomatic to write a block which evaluates {fluid.uiOptions.prefsEditor}.options.lazyLoad and then outputs the fluid.uiOptions.prefsEditor.lazyLoad grade if it is found.